WordPress/src/main/res/values/strings.xml

```diff
@@ -2356,6 +2356,14 @@
 
     <string name="google_error_timeout">Google took too long to respond. You may need to wait until you have a stronger internet connection.</string>
 
+    <!-- Post-Signup Interstitial -->
+    <string name="post_signup_interstitial_title">Welcome to WordPress</string>
+    <string name="post_signup_interstitial_subtitle">Whatever you want to create or share, we’ll help you do it right here.</string>
+    <string name="post_signup_interstitial_create_new_site">Create a new site</string>
+    <string name="post_signup_interstitial_add_self_hosted_site">Add a self-hosted site</string>
```

Comment:
Maybe it's possible to reuse some strings from existing screens. @SylvesterWilmott is currently considering this.

Reply:
We can reuse the string `Add self-hosted site` but let's keep `Create a new site`. In previous reviews `Create WordPress.com site` was voted too wordy.

Reply:
After some further discussion, I think we can also reuse `Create WordPress.com site` to be explicit about the button's function.

So the button strings should be:

`Create WordPress.com site`
`Add self-hosted site`

Reply:
I have now updated the strings to reuse existing ones from the **Site Picker** screen. I have also updated the screenshots to reflect this change.